st2client/st2client/commands/action.py

```diff
@@ -463,6 +463,8 @@ def _print_help(self, args, **kwargs):
             if action_ref_or_id:
                 try:
                     action = action_mgr.get_by_ref_or_id(args.ref_or_id, **kwargs)
+                    if not action:
```

Comment:
Yeah, I noticed some of those "raise" inconsistencies in the CLI - some of `get_` methods on the command class return None instead of throwing.